specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/networkWatcher.json

```diff
@@ -3380,6 +3397,248 @@
+        },
+        "protocol": {
+          "type": "string",
+          "description": "Network protocol.",
+          "enum": [
+            "Tcp",
+            "Http",
+            "Https",
+            "Icmp"
+          ],
+          "x-ms-enum": {
+            "name": "ConnectionMonitorTestConfigurationProtocol",
+            "modelAsString": true
+          }
+        },
+        "httpConfiguration": {
+          "$ref": "#/definitions/ConnectionMonitorHTTPConfiguration",
+          "description": "Describes HTTP configuration of connection monitor test."
+        },
+        "tcpConfiguration": {
+          "$ref": "#/definitions/ConnectionMonitorTCPConfiguration",
+          "description": "Describes TCP configuration of connection monitor test."
+        },
+        "icmpConfiguration": {
+          "$ref": "#/definitions/ConnectionMonitorICMPConfiguration",
+          "description": "Describes ICMP configuration of connection monitor test."
+        },
+        "healthThreshold": {
+          "$ref": "#/definitions/ConnectionMonitorHealthThreshold",
+          "description": "Describes health threshold of connection monitor test."
+        }
+      },
+      "required": [
+        "name",
+        "protocol"
+      ],
+      "description": "Describes the connection monitor test configuration."
+    },
+    "ConnectionMonitorTCPConfiguration": {
+      "properties": {
+        "enableTraceRoute": {
+          "type": "boolean",
+          "description": "A value indication whether trace route is enabled."
+        },
+        "port": {
+          "type": "integer",
+          "description": "The port to run the TCP probe."
+        }
+      },
+      "required": [
+        "port"
+      ],
+      "description": "Describes the TCP configuration."
+    },
+    "ConnectionMonitorICMPConfiguration": {
+      "properties": {
+        "enableTraceRoute": {
+          "type": "boolean",
+          "description": "A value indication whether trace route is enabled."
+        }
+      },
+      "description": "Describes the ICMP configuration."
+    },
+    "ConnectionMonitorHTTPConfiguration": {
+      "properties": {
+        "method": {
+          "type": "string",
+          "description": "HTTP method.",
+          "enum": [
+            "Get",
+            "Post"
+          ],
+          "x-ms-enum": {
+            "name": "HTTPConfigurationMethod",
+            "modelAsString": true
+          }
+        },
+        "port": {
+          "type": "integer",
+          "description": "The port to run the HTTP probe."
+        },
+        "path": {
+          "type": "string",
+          "description": "The path to add to provided URI."
+        },
+        "requestHeaders": {
+          "type": "array",
+          "description": "List of HTTP headers.",
+          "items": {
+            "$ref": "#/definitions/HTTPHeader"
+          }
+        },
+        "validStatusCodes": {
+          "type": "array",
+          "items": {
```

Comment:
Please add `description` for this property

Reply:
Added